src/vlt/src/read-password.ts

```diff
@@ -0,0 +1,41 @@
+import { error } from '@vltpkg/error-cause'
+
+export type Streams = {
+  stdin: NodeJS.ReadableStream & { setRawMode(mode: boolean): void }
+  stdout: NodeJS.WritableStream
+}
+
+export const readPassword = async (
+  prompt: string,
+  { stdin, stdout }: Streams = process,
+): Promise<string> => {
+  let input = ''
+  stdout.write(prompt)
+  stdin.setRawMode(true)
+  await new Promise<void>((res, rej) => {
+    stdin.on('data', (c: Buffer) => {
+      // backspace
+      if (c.length === 1 && c[0] === 0x7f) {
+        input = input.substring(0, input.length - 1)
+        stdout.write('\x1b[1D \x1b[1D')
+        return
+      }
+
+      input += String(c)
+      //eslint-disable-next-line no-control-regex
```

Comment:
i think this is a default eslint rule aimed at people not writing CLI applications. if you think its better, we can turn it off altogether?

Reply:
Personally, I feel like pretty much _all_ of the regex-related eslint rules are excessive and patronizing, but people also seem to think regular expressions are hard, and I think they're one of the most simple and straightforward features of JavaScript, so I try to accept my minority status and go along to get along. But yeah, we can definitely remove this one, definitely fine by me.